app/assets/javascripts/snowcrash/pages/nodes/new_form.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+  var copyOver = function($to, from, $typeTo, $typeFrom) {
+    if ($to.val() === '' && from !== '') {
+      $to.val(from.split('\n')[0] + '\n');
+      $typeTo.val($typeFrom.val());
+
+      // The click function we're in actually takes focus when clicked. If
+      // we called focus within the function it gets cancelled out. We give
+      // it a brief timout to allow it to occur after this function ends
+      // but seemingly at the same time.
+      setTimeout(function() {
+        $to.focus();
+      }, 0);
+    }
+  }
+
+  document.addEventListener("turbolinks:load", function() {
+    if ($('[data-behavior~=copy-node-label]').length) {
+      $('[data-behavior~=copy-node-label]').click(function(eventData) {
```

Comment:
`eventData` is not getting used anyway
```suggestion
      $('[data-behavior~=copy-node-label]').click(function() {
```